src/project/qml/MuseScore/Project/internal/CreateFromTemplatePage.qml

```diff
@@ -71,6 +71,7 @@ Item {
 
             onTitleClicked: function(index) {
                 model.currentCategoryIndex = index
+                templatePreview.load(model.currentTemplatePath)
```

Comment:
I would prefer if we fix this by making sure that `TemplatesModel` correctly emits the `currentTemplateChanged` signal when selecting another category.

Reply:
Yeah that sounds like a better idea